### ISSUE TITLE
Fehlende Kurse 

### ISSUE THREAD

**Issue by emmdie** [fsr/kpp#5]
Es fehlen mittlerweile einige Kurse. Insbesondere Studiengänge wie CMS sind nicht vertreten. Reicht es diese in /data/courses zu packen? Dann kann ich das auch fix machen. Hier eine auflistung des CMS-CE tracks:
[CMS_Course List_Komplexprüfungen.xlsx](https://github.com/fsr/kpp/files/10891924/CMS_Course.List_Komplexprufungen.xlsx)

**Comment by rouven0** [comment]
Ja, kannst die einfach hinzufügen. Würde es auch machen aber kann gerade keinen microsoft excel dateien öffnen
